src/Compiler/Microsoft.NET.Sdk.Razor.SourceGenerators.Transport/Microsoft.NET.Sdk.Razor.SourceGenerators.Transport.csproj

```diff
@@ -12,13 +12,18 @@
     <ExcludeFromSourceBuild>false</ExcludeFromSourceBuild>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis.ExternalAccess.RazorCompiler" ExcludeAssets="all" GeneratePathProperty="true" />
+  </ItemGroup>
```

Comment:
This file is more or less inlining the transitive reference. Rather than having a transitive reference it's just unpacking the NuPgk and re-packing the content. 

Why are we taking this approach here? Why doesn't a transitive reference work?

Reply:
I assumed that's the point of transport packages - they pack all the needed DLLs together, without relying on standard project/package references.